acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/steps/ContentSteps.java

```diff
@@ -266,30 +268,34 @@ public void iCanSeeAListOfPublicationsFromTheTimeSeries() throws Throwable {
             is(expectedPublicationTitles));
     }
 
-    // Scenario: Title validation ========================================================================
-    @When("^I populate the title with text which is too long$")
-    public void whenIPopulateTheTitleWithTextWhichIsTooLong() throws Throwable {
+    // Scenario: Title and Summary validation ========================================================================
+    @When("^I populate the (title|summary) with text which is ([0-9]+) characters long$")
```

Comment:
How about rephrasing this to say something along the lines of _'I populate the (title|summary) with text which exceeds maximum allowed limit of ([0-9]+) characters long'_?

I think this would restore the expressiveness of the original version of this step, while retaining ability to specify the actual number.